example/patches/nodejs-mobile-react-native+0.8.1.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/node_modules/nodejs-mobile-react-native/android/.project b/node_modules/nodejs-mobile-react-native/android/.project
```

Comment:
This patch is required as `nodejs-mobile-react-native` uses an old version of Gradle